flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliStrings.java

```diff
@@ -212,6 +212,9 @@ private CliStrings() {
 
     public static final String MESSAGE_BEGIN_STATEMENT_SET = "Begin a statement set.";
 
+    public static final String MESSAGE_NO_INSERT_STATEMENT_IN_STATEMENT_SET =
+            "No insert statement in statement set, skip submit.";
```

Comment:
```suggestion
            "No statement in the statement set, skip submit.";
```

I think we don't need to mention the statement kind, we may support more DML statements, e.g. `MERGE INTO` in the future.